uwuno/src/main/java/com/learning/uwuno/util/requestUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.learning.uwuno.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.learning.uwuno.errors.internalServerError;
+
+import java.util.Map;
+
+public final class requestUtil {
+    // Should not be instantiated
+    private requestUtil() {
+        throw new UnsupportedOperationException();
```

Comment:
Forgot to ask will this throw just kill the server and what error code will this return?

Reply:
It should never run because the constructor is private. I can remove it and just leave a comment instead.

Reply:
Oh if it never runs, I guess it doesn't matter.